Scheduler/observing_blocks_auxtel/AuxTel-PTC/AuxTel-PTC-3.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "auxtel_ptc_3",
+    "program": "auxtel_ptc_3",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "auxtel/take_image_latiss.py",
+            "standard": true,
+            "parameters": {
+              "program": "auxtel_ptc_3",
+              "reason": "auxtel_ptc_3",
+              "nimages": 20,
+              "filter": "SDSSg_65mm",
```

Comment:
For the darks/bias frames it will be safe to just use "empty_1" for the filter and grating. That way, if we ever take the "SDSSg_65mm" filter out, we won't need to update these.